lib/tasks/sidekiq_jobs.rake

```diff
@@ -10,4 +10,21 @@ namespace :sidekiq_jobs do
     AbstractJob.where(:created_at.lte => (Date.today - 7),
                       environment: 'preview').delete_all
   end
+
+  task :recover_interrupted_jobs do
+    p 'Recovering interrupted jobs ...'
+
+    harvest_jobs = HarvestJob.where(status: 'ready')
+
+    if harvest_jobs.count.zero?
```

Comment:
It might read nicer as:
```
if harvest_jobs.any?
  do things
else
  do no things
end
```

but really it's all good

Reply:
Yeah I can do that 👍